__test__/op-connect.test.ts

```diff
@@ -458,6 +458,43 @@ describe("Test OnePasswordConnect CRUD", () => {
             expect(otp).toEqual(OTP);
         });
     });
+
+    describe("List items by title contains", () => {
+        const title = "some title";
+        // Search all items by title which contains  the given value `title`
```

Comment:
```suggestion
```

Reply:
@ephremAddiss Please remove this comment. I think the test case description is self-descriptive enough.